SCClassLibrary/Common/Core/Boolean.sc

```diff
@@ -18,9 +18,13 @@ Boolean {
 	keywordWarnings {
 		// turn on/off warnings if a keyword argument is not found
 		_KeywordError
+		^this.primitiveFailed
+	}
+	trace {
+		// this is only available in a special debugging version of the app
```

Comment:
IMHO, for this PR we should keep it file-wide consistent and not change too much about non-PR-related issues like style etc.
This would IMHO only mess up the diff possibly needed for later corrections (fingers crossed)